package.json

```diff
@@ -9,15 +9,17 @@
   ],
   "scripts": {
     "develop": "gatsby develop",
-    "develop:secure": "dotenv -e .env.development -- gatsby develop --https -p=443",
-    "start": "gatsby develop",
+    "develop:secure": "gatsby develop --https -p=443",
+    "start": "yarn develop",
     "build": "gatsby build",
     "serve": "gatsby serve",
     "clean": "gatsby clean",
     "lint": "eslint .",
     "prettier": "prettier --check .",
     "format": "prettier --write .",
-    "test": "echo 'WARN: no test script set'",
+    "test": "LANG=en_US jest src --coverage && codecov",
```

Comment:
Some stuff are locale-dependent. Setting `LANG=en_US` ensures the app behaves consistently on all machines during test runs.